### ISSUE TITLE
Support retrieving from data from the API

### ISSUE THREAD

**Issue by coolgeek6667** [ramblingjordan/AbBOT-python#21]
There should be a command line option to pass `IP:Port` of an [AbBOT-api](https://github.com/ramblingjordan/AbBOT-api) instance to use for generating form data. If anything is specified, it would call the `/getFormBatch` endpoint to retrieve the data instead of using the standard tip body or the DeepAI endpoint that needs an api token.
Support for a token separated list would be beneficial to provide multiple API endpoint options, e.g., `--api 127.0.0.1:5000,192.168.1.18:5000,cloud-hosted-api.dev:5050`

**Comment by andria-dev** [comment]
This would be good. Are the extra API endpoints just fallbacks?

**Comment by coolgeek6667** [comment]
Yes, the extra API endpoints would just be fallbacks in case one goes down.
